src/daemon/utils_ssl.c

```diff
@@ -0,0 +1,100 @@
+/**
+ * collectd - src/utils_ssl.c
+ * Copyright (C) 2015  Toni Moreno
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; only version 2 of the License is applicable.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
+ *
+ * Author:
+ *   Toni Moreno <toni.moreno at gmail.com>
+ */
+
+ /* Code taken from The example code from Jeremy Brown:
+ * http://curl.haxx.se/libcurl/c/opensslthreadlock.html
+ **/
+
+#if HAVE_CONFIG_H
+# include "config.h"
+#endif
+
+# include "plugin.h"
```

Comment:
ups, you are right. I can not remember what I did here.. I will review ASAP and remove it if not needed